src/parser.yy

```diff
@@ -279,6 +279,24 @@ unit
     }
   | unit relation_decl {
         for (auto* cur : $relation_decl) {
+            if ((cur->getQualifier() & INPUT_RELATION) != 0) {
+                auto load = std::make_unique<AstLoad>();
+                load->addName(cur->getName());
+                load->setSrcLoc(cur->getSrcLoc());
+                driver.addLoad(std::move(load));
```

Comment:
I don't think the std::move is needed, here and below. If the compiler doesn't object, remove it.

Reply:
This cannot to be moved, otherwise it does not compile.

Reply:
Ah, that actually makes sense - if it was being returned it would be automatically moved, but that's not the case here.